modin/numpy/arr.py

```diff
@@ -485,71 +505,88 @@ def min(
         if self._ndim == 1:
             if axis == 1:
                 raise numpy.AxisError(1, 1)
-            result = self._query_compiler.min(axis=0)
```

Comment:
starting from here, it looks like a copy of `.max()` but with `min()` instead... why did we copy-paste so much?

I won't repeat my suggestions on `target = where.where(...) if foo else self` here, but they do apply

Reply:
A lot of the code is very similar with some small changes - I think @noloerino is planning a refactor for the future!